Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/infernal.py", line 182, in dafault_scan_for_riboswitches
    riboswitch_cmscan(fasta_path, out_dir)
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/infernal.py", line 155, in riboswitch_cmscan
    cmscan(
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/infernal.py", line 94, in cmscan
    ver = float(ver)
                ^^^
UnboundLocalError: cannot access local variable 'ver' where it is not associated with a value
